scripts/DocParser.js

```diff
@@ -29,8 +29,13 @@ const docVersionFile = `${dataDir}/docVersion.json`;
 const libraryDescriptionFile = `${dataDir}/libraryDescription.json`;
 const libraryDescription = {};
 
+// Documentation.js output is pruned for file size.  The following keys will be deleted:
+const KeysToIgnore = ['lineNumber', 'position', 'code', 'loc', 'context', 'path', 'loose', 'checked', 'todos', 'errors'];
+// These are allowed 'errors' in the documentation.  These are our custom tags.
+const AllowedErrorTags = ['@hoc', '@hocconfig', '@ui', '@required'];
```

Comment:
These are non-exhaustive, right? We have others we added I think.

Reply:
🤔 I think those are the only ones we've added that weren't already jsdoc tags.